.gitlab-ci.yml

```diff
@@ -26,61 +27,77 @@ init:
 RPM:
   stage: rpmbuild
   extends: .terraform
+  only:
+    - schedules
```

Comment:
That should be the opposite, only on PR/branch push, not on schedules.

Reply:
Yes, I was just trying something out. It's possible to also use `except` which in this case sounds better.